src/package.json

```diff
@@ -49,31 +49,31 @@
   },
   "homepage": "https://github.com/angular-schule/angular-cli-ghpages/#readme",
   "devDependencies": {
-    "@angular-devkit/architect": "^0.803.0",
-    "@angular-devkit/core": "^8.3.0",
-    "@angular-devkit/schematics": "^8.3.0",
-    "@types/fs-extra": "^8.0.0",
-    "@types/jest": "^24.0.18",
-    "@types/node": "^12.6.9",
-    "copyfiles": "^2.1.1",
-    "husky": "^3.0.7",
-    "jest": "^24.8.0",
-    "json-schema-to-typescript": "^7.0.0",
-    "prettier": "1.18.2",
-    "pretty-quick": "^1.11.1",
-    "rimraf": "^2.6.3",
-    "ts-jest": "^24.0.2",
-    "typescript": "^3.5.3"
+    "@angular-devkit/architect": "^0.1100.0",
+    "@angular-devkit/core": "^11.0.0",
+    "@angular-devkit/schematics": "^11.0.0",
+    "@types/fs-extra": "^9.0.4",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^14.14.7",
+    "copyfiles": "^2.4.0",
+    "husky": "^4.3.0",
+    "jest": "^26.6.3",
+    "json-schema-to-typescript": "^9.1.1",
+    "prettier": "2.1.2",
```

Comment:
That is the explanation! There must have been changes in the rules.

Reply:
Ah Yeah. That might be related!